develop/tutorials/articles/100-tooling/02-liferay-ide/10-searching-liferay-portal-source-in-liferay-ide.markdown

```diff
@@ -25,6 +25,17 @@ advanced search. See the
 [Managing the Target Platform in Liferay Workspace](/docs/7-0/tutorials/-/knowledge_base/t/managing-the-target-platform-for-liferay-workspace)
 tutorial for more information on how this works.
 
+Workspace does not perform portal source indexing by default. You must enable
+this functionality by adding the following property to your workspace's
+`gradle.properties` file:
+
+```properties
+target.platform.index.sources=true
+```
+
+| **Note:** The disablement of portal source indexing is disabled in Gradle
```

Comment:
The disablement is disabled? :-) 

Can we just say portal source indexing is enabled? This would apply for all the branches.